src/odemis/driver/tmcm.py

```diff
@@ -2778,12 +2781,12 @@ def MoveAbsPos(self, axis, pos):
         axis (0<=int<=5): axis number
         pos (-2**31 <= int 2*31-1): position
         """
-        self._node.controlword = 0x0f  # switch on
         self.SetAxisParam(axis, TARGET_POSITION, pos)
         # Bit 6 of controlword: absolute (0) or relative (1)
         # Bits 0-3 to 1: switch on and enable (see p 71 of canopen firmware manual)
         # Bit 4: start positioning
-        self._node.controlword = 0x1f
+        self._node.controlword = 0b0001111  # switch on
+        self._node.controlword = 0b0011111
```

Comment:
Please add the comment about double setting the value also here (or just put a "cf MoveRelPos").